src/assets/thirdPartyServices/mozi/gene_annotation_service/result/index.js

```diff
@@ -137,6 +242,8 @@ const AnnotationResult = props => {
           handleClose={() => setTableShown(false)}
         />
       )}
+      {response && console.log("response.csv_files", response.csv_files)}
```

Comment:
remove this console.log

Reply:
Thank you, i have removed all console logs